server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -678,4 +678,15 @@ public void fixingReIndexingShouldNotFailWhenNoMail() {
             .body("type", is("ErrorRecoveryIndexation"));
     }
 
+    @Test
+    public void postRedeliverAllEventsShouldCreateATask() {
```

Comment:
does it test that the task is serialized?

Reply:
yes, but anyway I will improve the tests to be more coherent